lib/directives/tree-drag.directive.ts

```diff
@@ -22,10 +22,17 @@ export class TreeDragDirective implements DoCheck {
   @HostListener('dragstart', ['$event']) onDragStart(ev) {
     // setting the data is required by firefox
     ev.dataTransfer.setData('text', ev.target.id);
+
+    if (this.draggedElement.mouseAction) {
```

Comment:
It works but if you check `this.treeDraggedElement` in the event handler you won't get the updated dragged element.
I don't remember why the timeout is there